server/integration/feature-pack/pom.xml

```diff
@@ -528,6 +528,11 @@
             <artifactId>netty-transport-native-epoll</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-unix-common</artifactId>
```

Comment:
why do we need this?

Reply:
might make more sense to pull in the bom?

Reply:
The Enforcer plugin discovered another transitive dependency of `netty-transport-native-epoll` IIRC. Yes Netty is *fragmented*, not *modular*.

Reply:
@johnou Is this a kitchensink import? We want to include only what's needed.

@slaskawi I see, it's a new module in 4.1.11+

Reply:
@rvansa You probably mean `9.1.11`. Yes.

Reply:
@rvansa no, as bom's are defined in dependencyManagement only the metadata is pulled in, you still need to define every dependency you want to use but you can omit the version.

See https://github.com/netty/netty/issues/5994#issuecomment-261815093 for a good explanation.

Reply:
@johnou OK, thanks for explanation. Yes, in this case importing the bom would make sense. @slaskawi Would you like to include that change here?

@slaskawi No, I mean new module in netty 4.1.11.

Reply:
@slaskawi if you are using native transports you might want to wait until the next Netty release, see https://github.com/netty/netty/pull/7563

Reply:
> @johnou Is this a kitchensink import? We want to include only what's needed.

dependencyManagement != dependencies

Whatever goes into dependencyManagement is not dependent upon necessarily. You have to then select what you depend on. We encourage users to do that all the time with Infinispan's bom ([here](https://github.com/infinispan/infinispan-simple-tutorials/blob/master/pom.xml#L14-L24)).

Reply:
I agree with you guys - let's use the BOM. However I wasn't able to do a full cleanup. The Karaf feature from `persistence/remote` pull in `client/hotrod`, which in turn uses Maven filtering to fill out its `features.xml` file. I believe this can be fixed by using Netty as bundles and not wrapping them. I created [ISPN-8735](https://issues.jboss.org/browse/ISPN-8735) to investigate this option. Any volunteers?